app/components/channel_intro/index.js

```diff
@@ -37,28 +25,22 @@ function makeMapStateToProps() {
         }
     );
 
-    const getChannelMembersForDm = createSelector(
-        (state, channel) => getUser(state, getOtherUserIdForDm(state, channel)),
-        (otherUser) => {
-            if (!otherUser) {
-                return [];
-            }
-
-            return [otherUser];
-        }
-    );
-
     return function mapStateToProps(state, ownProps) {
         const currentChannel = getChannel(state, {id: ownProps.channelId}) || {};
         const {status: getPostsRequestStatus} = state.requests.posts.getPosts;
 
+        const teammate = [];
```

Comment:
I really want to revert the changes here as we should not use `channel.teammate_id` I've explained the reason above